tensorflow/core/grappler/optimizers/auto_mixed_precision.h

```diff
@@ -49,6 +50,9 @@ class AutoMixedPrecision : public GraphOptimizer {
         return "auto_mixed_precision_onednn_bfloat16";
       case AutoMixedPrecisionMode::CPU:
         return "auto_mixed_precision_cpu";
+      case AutoMixedPrecisionMode::FP16_CPU:
+        // Note: use same config for FP16 on CPU & GPU.
+        return "auto_mixed_precision";
```

Comment:
I tested with "auto_mixed_precision_onednn_float16" here and dumped files. It does make it more clear, it will help for dump and debugging to have a distinguishing name while still use the same config.